docs/admin/open-in-coder.md

```diff
@@ -0,0 +1,29 @@
+# Open in Coder Button
+
+Add a markdown button to your project's `README.md` to get your developers up and running with Coder with a few clicks.
+
+A basic example looks like this:
+
+```text
+[![Open in Coder](https://cdn.coder.com/embed-button.svg)](https://dev.coder.com/?template=coder-ts)
```

Comment:
Why use a query parameter instead of just using `/templates/<name>`?

Reply:
I don't think we had originally considered it. The current approach doesn't care about the `/templates` path from the API the user interacts with. One downside would be that we're tying it to the templates path and if we change it, we'd have to support the old approach and the new approach.

Probably not a huge deal and your suggestion makes more sense considering our original [assumptions](https://github.com/coder/coder/issues/3981#issuecomment-1247186536). 

Should I change it?

Reply:
I think changing it would make sense!

Reply:
Agree with changing it to `/templates/<name>`

Reply:
I'm a bit confused about what this PR does given links to `/templates/<name>` can be used to go to the template page.

Reply:
Cool! I agree, makes sense to change.

> I'm a bit confused about what this PR does given links to /templates/<name> can be used to go to the template page.

Guess I'm confused too 😂 @bpmct I thought we agreed to <deployment-url> + <query params> in [our original spec](https://github.com/coder/coder/issues/3981#issuecomment-1247186536)?

Reply:
Yeah - a good learning experience for me, my bad.

In v1, we have a separate path/flow for "Open in Coder" so it's not necessarily a problem that it is on a separate path, but I don't think we should stick to an original spec when a better route cames up, even if it results in some churn. The reason it was on a separate path for v1 was to handle Git OAuth a / different flow which wasn't addressed here. If we anticipate we'll do that down the road, I can see the merit of doing it that way.

Secondly, in its current state, the feature felt more like debt versus a UX improvement since the same value can be provided as a link to the templates page. If this provided additional functionality and a second route was necessary, I think it would have made more sense to keep.

Reply:
Ah, that make sense. 

No worries at all! I agree, using the templates path makes more sense here and won't introduce any debt. I don't want swaying from the spec. Just wanted to make sure I didn't miss anything and also making sure we're on the same page 👍🏼 I'll make these changes then re-request a review.